docs/standard/index.md

```diff
@@ -14,65 +14,48 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
+You can also read about [.NET Architectural Components](components.md) to get an overview of the various "pieces" of .NET and how they fit together.
 
-Multiple [.NET implementations and products](components.md) are available, based on open [.NET Standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) that specify the fundamentals of the platform. They are separately optimized for different application types (for example, desktop, mobile, gaming, cloud) and support many chips (for example, x86/x64, ARM) and operating systems (for example, Windows, Linux, iOS, Android, macOS). Open source is also an important part of the .NET ecosystem, with multiple .NET implementations and many libraries available under OSI-approved licenses.
+## New to .NET Core
 
-- Learn about [C#](../csharp/index.md)
-- Learn about [F#](../fsharp/index.md)
-- Browse the [.NET API Library](../../api/index.md)
-- [Introduction to the Common Language Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/intro-to-clr.md)
+If you're new to .NET Core, check out [Getting Started with .NET Core](../core/getting-started.md).
 
-Fundamentals
-------------
+## New to .NET Standard
 
-**Multi-Language** -- .NET provides a well-defined type system, file formats, runtime, framework and tools that can be used by multiple languages, both for their own execution and also to interoperate with other languages using those same components of .NET as their shared currency.
+If you're new to .NET Standard, check out [.NET Standard Library](library.md).
 
-**Managed Memory** -- .NET automatically manages memory for you via a garbage collector. It ensures that you always reference live objects, guaranteeing that you avoid nasty problems like buffer overruns and access violations. This includes array bounds checking.
+## Porting .NET Framework Code to .NET Core
 
-**Type Safety** -- The primary .NET model for functionality and memory representation is "types". Types define shape and optionally behavior. The runtime ensures that calling code can only operate on types according to their definition and specified visibility of members, providing consistent, reliable and secure results.
+If you're looking to port an application, service, or some component of a system to .NET Core, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
 
-Features
---------
+## Porting a NuGet package from to .NET Standard
 
-**User-defined Value Types** -- Value types are a useful category of types since they offer the semantic of "pass by value" instead of "pass by reference", as is the case for classes. Value types are most obviously usefully for numeric data. .NET enables value types for both primitive types, like integers, and user-defined types.
+If you're looking to port a NuGet package to .NET Standard, check out [Porting to .NET Core from .NET Framework](../core/porting/index.md).
```

Comment:
Yeah, right now the article is itself about porting to .NET Core, but is also relevant for .NET Standard.  This is something we'll have to clean up later, but for now I can update the text saying that the concerns are shared here.